benchexec/tools/veriabs.py

```diff
@@ -46,7 +46,7 @@ def name(self):
         return "VeriAbs"
 
     def project_url(self):
-        return "https://www.tcs.com/designing-complex-intelligent-systems"
+        return "https://zenodo.org/records/10243500"
```

Comment:
This refers to a specific version of VeriAbs, so this URL will be wrong for all benchmarks done with a different version of VeriAbs, and thus could lead to confusion. Don't you have a project page with information about this tool that could be used here?

If you really wish to link to Zenodo, then at least the meta DOI that always points to the latest version should be used.